<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RequeteWindow</class>
 <widget class="QDialog" name="RequeteWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Effectuer une requête</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Sujet de recherche</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Que voulez-vous rechercher ?</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QRadioButton" name="radioInteractions">
              <property name="text">
               <string>&amp;Interactions</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioTodos">
              <property name="text">
               <string>&amp;Todos</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioDatesTodo">
              <property name="text">
               <string>&amp;Dates de todo</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Quel est le contact concerné ?</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxContact">
            <property name="editable">
             <bool>true</bool>
            </property>
            <property name="insertPolicy">
             <enum>QComboBox::NoInsert</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Filtres</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QFrame" name="frameFiltresInteraction">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Rechercher les interactions dont la date de création est entre :</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QGridLayout" name="gridLayout">
               <item row="1" column="0">
                <widget class="QDateEdit" name="dateEditInteractionMax">
                 <property name="calendarPopup">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QDateEdit" name="dateEditInteractionMin">
                 <property name="calendarPopup">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QCheckBox" name="checkBoxInteractionDateMin">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="toolTip">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Si désactivé, alors seules les interactions dont la date est inférieure à la date maximale spécifiée seront extraites.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QCheckBox" name="checkBoxInteractionDateMax">
                 <property name="toolTip">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Si désactivé, alors seules les interactions dont la date est supérieure à la date minimale spécifiée seront extraites.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frameFiltresTodo">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_10">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Rechercher les todos dont la date de réalisation est entre :</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="1" column="0">
                <widget class="QDateEdit" name="dateEditTodoMax">
                 <property name="calendarPopup">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QDateEdit" name="dateEditTodoMin">
                 <property name="calendarPopup">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QCheckBox" name="checkBoxTodoDateMin">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="toolTip">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Si désactivé, alors seuls les todos dont la date est inférieure à la date maximale spécifiée seront extraits.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QCheckBox" name="checkBoxTodoDateMax">
                 <property name="toolTip">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Si désactivé, alors seuls les todos dont la date est supérieure à la date minimale spécifiée seront extraits.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Résultat(s) de la requête :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textEditResultats">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="acceptRichText">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string>Aucun résultat.</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
       <item>
        <widget class="QPushButton" name="buttonRechercher">
         <property name="text">
          <string>&amp;Rechercher</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonFermer">
         <property name="text">
          <string>&amp;Fermer</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonFermer</sender>
   <signal>clicked()</signal>
   <receiver>RequeteWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>546</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>559</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioInteractions</sender>
   <signal>toggled(bool)</signal>
   <receiver>frameFiltresInteraction</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>29</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>35</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioTodos</sender>
   <signal>toggled(bool)</signal>
   <receiver>frameFiltresTodo</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioDatesTodo</sender>
   <signal>toggled(bool)</signal>
   <receiver>frameFiltresTodo</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxInteractionDateMin</sender>
   <signal>toggled(bool)</signal>
   <receiver>dateEditInteractionMin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxInteractionDateMax</sender>
   <signal>toggled(bool)</signal>
   <receiver>dateEditInteractionMax</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxTodoDateMin</sender>
   <signal>toggled(bool)</signal>
   <receiver>dateEditTodoMin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxTodoDateMax</sender>
   <signal>toggled(bool)</signal>
   <receiver>dateEditTodoMax</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>348</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
